agent/php_agent.h

```diff
@@ -133,13 +133,26 @@ typedef unsigned int nr_php_object_handle_t;
  * Purpose : Extract the named entity from a PHP object zval.
  *
  * Params  : 1. The object to extract the property from.
- *           2. The name of the object.
+ *           2. The name of the object property.
  *
  * Returns : The specified element or NULL if it was not found.
  */
 extern zval* nr_php_get_zval_object_property(zval* object,
                                              const char* cname TSRMLS_DC);
 
+/*
+ * Purpose : Extract the named entity from a PHP exception zval.
+ *           This determines if the passed exception is a PHP error
+ *           or a PHP exception, and extracts accordingly.
+ *
+ * Params  : 1. The exception to extract the property from.
+ *           2. The name of the exception property.
+ *
+ * Returns : The specified element or NULL if it was not found.
```

Comment:
Who owns returned `zval*`? I.e.: does it need to be freed by the caller?

Reply:
added a clarifying comment in [34e9324](https://github.com/newrelic/newrelic-php-agent/pull/825/commits/34e932403b8b9ac8e8543188a582cf5c9e2d8ddd)